src/index.js

```diff
@@ -209,7 +208,11 @@ class Masq {
     return new Promise((resolve, reject) => {
       // Subscribe to channel for a limited time to sync with masq
       debug(`Creation of a hub with ${channel} channel name`)
-      const hub = signalhub(channel, this.config.hubUrls)
+      const hubErrorHandler = (error) => {
+        reject(new MasqError(MasqError.SIGNALLING_SERVER_ERROR, error))
+      }
+
+      const hub = signalhub(channel, this.config.hubUrls, undefined, hubErrorHandler)
```

Comment:
any reason to not use an `EventEmitter` to emit the error from signalhubws ?

We could do : `hub.on('error', err => ())`